frontend/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "frontend",
```

Comment:
Can you specify here which is the target node and npm version. See: https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json

Most likely we want `node >= 18.0.0 < 19.0.0`.